webportal/src/app/cluster-view/hardware/hardware-detail.component.js

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation
+// All rights reserved.
+//
+// MIT License
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+// documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+// the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+// to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+// The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+// BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+//
+
+const hardwareDetailComponent = require('./hardware-detail.component.ejs');
+const breadcrumbComponent = require('../../job/breadcrumb/breadcrumb.component.ejs');
+const webportalConfig = require('../../config/webportal.config.json');
+const url = require('url');
+
+//
+
+const resizeContentWrapper = () => {
+  $('#content-wrapper').css({'height': $(window).height() + 'px'});
+  $('#content-iframe').css('height', (($(window).height() - 120)) + 'px');
```

Comment:
What does the magic "120" means?

Reply:
It is the number of pixels from the top of the page to the top of the iframe -- I haven't found a solution to make the iframe automatically fit into the space.

Reply:
Not looks perfect, but let's move on first.

Reply:
Agree. Will find a way eventually.